driver-core/src/main/java/com/datastax/driver/core/ArrayBackedRow.java

```diff
@@ -25,6 +26,13 @@
  */
 class ArrayBackedRow extends AbstractGettableData implements Row {
 
+    /**
+     * A pattern to parse (non-aliased) token column names of the form token(x).
+     * Note that starting from Cassandra 2.2 built-in functions are declared in
+     * the system keyspace, so the function name is prefixed with "system.".
+     */
+    private static final Pattern TOKEN_COLUMN_NAME = Pattern.compile("(system\\.)?token(.*)");
```

Comment:
This is due to the fact that now built-in function names are returned with the system prefix.